eZ/Publish/API/Repository/Tests/FieldType/KeywordIntegrationTest.php

```diff
@@ -395,8 +395,10 @@ private function assertContentFieldHasCorrectData($contentId, KeywordValue $valu
     public function testKeywordsAreCaseSensitive()
     {
         $contentType = $this->testCreateContentType();
-        $publishedContent01 = $this->createAndPublishContent('Foo', $contentType, md5(uniqid() . microtime()));
-        $publishedContent02 = $this->createAndPublishContent('foo', $contentType, md5(uniqid() . microtime()));
+        /** @noinspection NonSecureUniqidUsageInspection */
+        $publishedContent01 = $this->createAndPublishContent('Foo', $contentType, md5(uniqid('', true)));
```

Comment:
Usually these warnings are for some reason. Are you sure? :)

Reply:
Pretty sure :) This is a test, unlikely to hit any concurrency problem, and there's no security problem. But we could of course add a prefix, so we can remove the tag.

Reply:
Still, I don't see any harm in using prefix, it's even less code (depending on a prefix). You could just use `__METHOD__` for prefix if nothing better fits here.

Reply:
Sure thing. 👍